tests/integration/cairo_compiler/prepare_files_fixture.py

```diff
@@ -0,0 +1,61 @@
+from typing import Callable
+from pathlib import Path
+from enum import Enum
+
+from tests.integration._conftest import ProtostarFixture
+
+TEST_CONTRACTS_PATH = Path(__file__).parent / "contracts"
+
+
+class RequestedFiles(Enum):
+    input_enum_contract_cairo = 1
+    input_basic_starknet_contract_cairo = 2
+    input_basic_starknet_test_cairo = 3
+    input_roll_test_cairo = 4
+    output_sierra = 5
+    output_casm = 6
```

Comment:
You can use enum with string values here

Reply:
And all the `if` statements from `prepare_files` can be removed then :)

Reply:
[use string based enum](https://github.com/software-mansion/protostar/pull/1316/commits/7b57752e0b001ee3892304d29b621bb5e070154e)